pages/install/firefish.md

```diff
@@ -1,16 +1,37 @@
 ---
 title: Fossil Gen 4
-deviceName: firefish
+deviceNames: [firefish, ray]
 label: ray/firefish
 section: install
-layout: aw-install
+layout: aw-install-multi
 ---
 
+<div class="callout callout-warning">
+    <h4>Variants</h4>
+    <p>The Fossil Gen 4 is marketed as different versions from different manufacturers.
+    This means that there are two known versions that are compatible with a range of Fossil Gen 4 watches.
+    These two versions will be referred to as: <code>ray</code> and <code>firefish</code></p>
+    <code>firefish</code> is compatible with the following variants:
+    <ul>
+       <li>Fossil Gen 4 46mm</li>
+       <li>Fossil Q Explorist HR</li>
+       <li>Diesel On Full Guard 2.5</li>
+       <li>Misfit Vapor 2 (46mm)</li>
+    </ul>
+    <code>ray</code> is compatible with the following variants:
+    <ul>
+       <li>Fossil Gen 4 (41mm)</li>
+       <li>Skagen Falster 2</li>
+       <li>Fossil Q Venture HR</li>
+       <li>Misfit vapor 2 (41mm)</li>
+    </ul>
+</div>
 
```

Comment:
there's a few more models, such as the `michael kors access runway` (which seems to be a ray) and some models from guess connect that I can't confirm the names of (the current source for this is the wearos spec spreadsheet, but it's not reliable)
Imo it's worth saying that other models may also be compatible. There's a lot of them and information on some of them is rather flaky.
It might also be worth mentioning that firefish generally has model number DW6xx and ray has DW7xx, as a way of identifying whether the user's watch is a ray or a firefish (most of the watches' names are only printed on the box and the caseback only has the brand name so are hard to identify)

Reply:
I've added the Michael Kors Access Runway and removed the Fossil Gen 4 lines as they are redundant with the other Fossil models.

Reply:
Also added the DW6xx and DW7xx.